client/pages/AssessmentHome/FraHome/FraHome.tsx

```diff
@@ -54,21 +55,24 @@ const FraHome: React.FC = () => {
           ))}
         </div>
       )}
-
-      <Switch>
-        <Redirect
-          from={BasePaths.Assessment.home()}
-          to={BasePaths.Assessment.home(countryIso, assessmentName, cycleName, AssessmentHomeRouteNames.overview)}
-          exact
-        />
-        {sections.map(({ name, component }) => (
-          <Route
-            key={name}
-            path={BasePaths.Assessment.home(countryIso, assessmentName, cycleName, name)}
-            component={component}
+      {displayTabs ? (
```

Comment:
it was already handled. now there's no need to add this. Dashboard is always added. leave it as it was

Reply:
check useSections